src/Stats/Job.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Amp\Beanstalk\Stats;
+
+class Job {
+    use Initializer;
+
+    public $id;
+
+    public $tube;
+
+    public $state;
+
+    public $pri;
+
+    public $age;
+
+    public $delay;
+
+    public $ttr;
+
+    public $time_left;
```

Comment:
I prefer camelCase here for consistency, but it will have a slight impact on the initializer, but it's not a bottleneck.